src/book.py

```diff
@@ -413,6 +413,18 @@ def _read_coinbase(self, file_path: Path) -> None:
                     # If it's a buy, add the corresponding sell to complement
                     # the trading pair.
                     elif operation == "Buy":
+                        # If coins were bought with fiat, first add a corresponding deposit
+                        if misc.is_fiat(_currency_spot):
+                            eur_total = misc.force_decimal(_eur_total)
+                            self.append_operation(
+                                "Deposit",
+                                utc_time,
+                                platform,
+                                eur_total,
+                                _currency_spot,
+                                row,
+                                file_path,
+                            )
```

Comment:
As it's also possible to deposit fiat first and buy later on, we have to check against that and can not merely add a deposit action.

It might be ok to disable tracking of deposit and withdrawals with home fiat which might solve most of the problem.
We can add a config variable for that. and should raise a warning when we adjust the config variable while evaluating the coinbase account statement, which as a buy with fiat.

If it is still bothering we should add a file in parallel which lists all buys to coinbase and initialize there value as 0 (do not add deposit on default). The user can adjust the file so that deposits will be added on the second run.

The format of the could be ini (csv might be a good "normal" format, but editing csv with Excel fucks up the format most of the time which might create a headache for most of the users) and should be saved in the account statements folder.